Makefile

```diff
@@ -65,9 +65,8 @@ test: $(SOURCE) VERSION .gobuild
 	    $(TEST_COMMAND)
 
 lint:
-	for source_file in $(SOURCE); do \
-		go vet -x $$source_file ; \
-	done
```

Comment:
`golint` is not part of the golang docker image, sadly. With regards to linting, my reckoning was to make sure it was part of the CI system, over running it locally.